.github/actions/GetGitBranches/action.ps1

```diff
@@ -0,0 +1,25 @@
+param (
+    [Parameter(Mandatory = $false, HelpMessage="JSON-formatted array of branch names to include")]
+    [string] $include = '[]'
+)
+
+$includeBranches = ConvertFrom-Json $include
+
+git fetch
```

Comment:
consider RunAndCheck command for git operations, to check on exit code

Reply:
Done.